pom.xml

```diff
@@ -43,6 +45,12 @@
             <groupId>com.vaadin</groupId>
             <!-- Replace artifactId with vaadin-core to use only free components -->
             <artifactId>vaadin</artifactId>
+            <exclusions>
```

Comment:
No exclusion of hilla? Wouldn't this result in spring dependencies added to a plain jakarta project?